### DISCUSSION TITLE
Langfuse Roadmap on GitHub Discussions

### DISCUSSION THREAD

**Discussion by marcklingen** [langfuse/langfuse#1396] [Announcements]
Hi all,

Langfuse is developing quickly and we want to be super transparent what we are working on and what's next. That's why we added a public roadmap which we'll update continuously: https://langfuse.com/roadmap

Looking forward to your feedback and thoughts (in this thread).

Other great ways to shape the roadmap
- Submit and vote on [ideas](https://github.com/orgs/langfuse/discussions/categories/ideas)
- Raise [issues](https://github.com/langfuse/langfuse/issues) for any bugs

Thank you!
@maxdeichmann, @clemra, and @marcklingen (authors/maintainers)

**Comment by ankitm10039** [comment]
Hi All,
Can we directly call Langfuse reference APIs? Currently, we are using Langfuse only for internal function logging. If this approach is feasible, please provide some guidance or best practices so that I can implement it correctly.

Also, could you share details on:

Which SDKs we are using for Langfuse integration (Frontend / Backend)?

The overall application architecture or flow for Langfuse setup?

This will help ensure that the implementation is consistent and aligned with the existing structure.

**Comment by Amjath73** [comment]
Hi All,
Everything was working fine when using langfuse locally but suddenly it started it stopped the connection with my project, and i was not able to get traces. I am using postgres.
"Failed to upload JSON to S3".

Anyone have any idea on this issue?
